book/routing.rst

```diff
@@ -266,10 +189,13 @@ A basic route consists of just two parts: the ``path`` to match and a
 
         return $collection;
 
-This route matches the homepage (``/``) and maps it to the
-``AppBundle:Main:homepage`` controller. The ``_controller`` string is
-translated by Symfony into an actual PHP function and executed. That process
-will be explained shortly in the :ref:`controller-string-syntax` section.
+Without much explanation (it is coming), when client visits URL like
+``http://example.com/`` it matches to ``/`` which executes the appropriate
+controller which returns a ``Response`` object.
+
+Of course the routing system supports much more interesting routes. So,
+to a bit more advanced examples which will explain how the routing works in
+detail.
```

Comment:
I like this idea, how about:

>
But the routing system supports routes that are much more flexible than this! Time to
dive deeper into the routing system.